tor/core/user_interaction.py

```diff
@@ -268,7 +279,7 @@ def process_done(
         elif done_response.status == BlossomStatus.blacklisted:
             message = i18n["responses"]["general"]["blacklisted"]
 
-    return message, return_flair
+    return message, return_flair, alt_promotion_txt
```

Comment:
This will not work, as only two arguments are expected to be returned. Instead, we will have to modify the `message` variable to append the promotion text.
We could do something similar to the `alt_text_trigger` in line 270, e.g.

```py
message = f"{message}\n\n{alt_promotion_txt}"
```